tools/crm_mon.c

```diff
@@ -2070,7 +2091,7 @@ refresh_after_event(gboolean data_updated, gboolean enforce)
         refresh_timer = mainloop_timer_add("refresh", 2000, FALSE, mon_trigger_refresh, NULL);
     }
 
-    if (reconnect_timer > 0) {
+    if (!enforce && (reconnect_timer > 0)) {
```

Comment:
This gets us around the early return so that we set the `refresh_trigger`, calling `mon_refresh_display()`.

Reply:
the idea is if we're about to reconnect, there's no point in refreshing yet. but if we need to display some state info in the meantime, something else might be appropriate